scripts/lintcommit.lua

```diff
@@ -0,0 +1,131 @@
+function commit_message_is_ok(commit_message)
+  local commit_split = vim.fn.split(commit_message, ":")
```

Comment:
Could also use `vim.split` here and elsewhere.

Reply:
I need a *little* bit more context :). Any reason to choose `vim.split` over `vim.fn.split`?

Reply:
> I need a *little* bit more context :). Any reason to choose `vim.split` over `vim.fn.split`?

Every call to `vim.fn...` calls a vim function using the API, therefore conversion of values from `Lua -> API -> VimL` and then reverse. Some functions have been re-implemented in Lua.

I think that's the reason

Reply:
:pray: